tests/components/dexcom/test_sensor.py

```diff
@@ -0,0 +1,91 @@
+"""The sensor tests for the griddy platform."""
+from asynctest import patch
+from pydexcom import GlucoseReading
+
+from homeassistant.components.dexcom.const import DOMAIN
+from homeassistant.const import (
+    CONF_PASSWORD,
+    CONF_UNIT_OF_MEASUREMENT,
+    CONF_USERNAME,
+    STATE_UNAVAILABLE,
+)
+from homeassistant.setup import async_setup_component
+
+
+async def test_sensors(hass):
+    """Test we get sensor data."""
+    _mock_glucose_reading = GlucoseReading(
+        {
+            "DT": "/Date(1587165223000+0000)/",
+            "ST": "/Date(1587179623000)/",
+            "Trend": 4,
+            "Value": 110,
+            "WT": "/Date(1587179623000)/",
+        }
+    )
+    with patch(
+        "homeassistant.components.dexcom.Dexcom.get_current_glucose_reading",
+        return_value=_mock_glucose_reading,
+    ), patch(
+        "homeassistant.components.dexcom.Dexcom.create_session",
+        return_value="test_session_id",
+    ):
+        assert await async_setup_component(
+            hass,
+            DOMAIN,
+            {
+                DOMAIN: {
+                    CONF_USERNAME: "test_username",
+                    CONF_PASSWORD: "test_password",
+                    "server": "US",
+                    CONF_UNIT_OF_MEASUREMENT: "mg/dL",
+                }
+            },
+        )
+        await hass.async_block_till_done()
+    sensor_dexcom_test_username_glucose_value = hass.states.get(
```

Comment:
Please shorten this variable name. It doesn't have to be this long. We know that we're testing the dexcom integration sensor platform.